src/sentry/static/sentry/app/actionCreators/streamTag.jsx

```diff
@@ -0,0 +1,29 @@
+import {Client} from '../api';
+import {t} from '../locale';
+import StreamTagStore from '../stores/streamTagStore';
+import StreamTagActions from '../actions/streamTagActions';
+import AlertActions from '../actions/alertActions';
+
+const api = new Client();
+
+const MAX_TAGS = 500;
+
+export function fetchStreamTags(orgId, projectId) {
+  StreamTagStore.reset();
+  StreamTagActions.loadTags();
+
+  api.request(`/projects/${orgId}/${projectId}/tags/`, {
+    success: tags => {
+      let trimmedTags = tags.slice(0, MAX_TAGS);
+
+      if (tags.length > MAX_TAGS) {
+        AlertActions.addAlert({
```

Comment:
I'm not sure, but would indicator make more sense here?

Reply:
Not sure, just copy and pasted this and don't want to change this for now since it looks quite different